tgui/packages/tgui/interfaces/Phone.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @file
+ * @copyright 2022
+ * @author DisturbHerb (https://github.com/disturbherb)
+ * @license MIT
+ */
+
+import { useBackend } from '../backend';
+import { Button, Collapsible, Dimmer, Section } from '../components';
+import { Window } from '../layouts';
+
+export const Phone = (props, context) => {
+  const { data } = useBackend(context);
+  const { inCall, dialing, name } = data;
+  return (
+    <Window title={name} width={250} height={350}>
+      <Window.Content>
+        {(dialing || inCall) && (
+          <Dimmer>
+            <h1>LINE BUSY</h1>
+          </Dimmer>
+        )}
+        <Section title="Phonebook" fill scrollable>
+          <AddressGroup category="bridge" name="Bridge" depColour="green" />
```

Comment:
I feel like we probably have department colors in variables somewhere? Possibly in the SCSS stuff, in which case maybe apply a class rather than a prop?